cg/meta/report/rnafusion.py

```diff
@@ -55,7 +47,9 @@ def get_sample_metadata(
             input_amount=self.lims_api.get_latest_rna_input_amount(sample_id=sample.internal_id),
             insert_size=None,
             insert_size_peak=None,
-            mapped_reads=sample_metrics.reads_aligned / sample_metrics.before_filtering_total_reads,
+            mapped_reads=sample_metrics.reads_aligned
```

Comment:
Should reads_aligned be renamed to reflect that it is read pairs and not single reads - to avoid future confusion.

Reply:
these are standard names given by multiqc. So I don't think its a great idea to change them. It is nice that the metrics collected by multiqc always get assigned the same names regardless of pipeline, etc. I do agree that several of these names are confusing. But maybe something to think about in the future? I don't think I would do it now

Reply:
Ok, let's leave it for now 👍. If you want to you could make an issue on multiqc to clarify the variable name.